core/builtins/parser/command.py

```
@@ -50,16 +50,16 @@ def __init__(
+            if match.command_template:
+                for m in match.command_template:
+                    command_templates[m] = {"priority": match.priority, "meta": match}
             else:
-                help_docs[""] = {"priority": match.priority, "meta": match}
+                command_templates[""] = {"priority": match.priority, "meta": match}
             if match.options_desc:
                 for m in match.options_desc:
                     self.options_desc[m] = match.options_desc[m]
 
-        self.args: Dict[Union[Template, ""], dict] = help_docs
+        self.args: Dict[Union[Template, ""], dict] = command_templates
```

Ruff (0.14.1)

Syntax error in forward annotation: Expected an expression

(F722)